contents/blog/best-survicate-alternatives.mdx

```diff
@@ -0,0 +1,436 @@
+
+- 💬 **Surveys:** Target surveys by event or user properties. Templates for [net promoter score (NPS)](/templates/nps-survey), [product-market fit (PMF)](/templates/pmf-survey) surveys, and more.
+
+- 📈 **Product analytics:** Custom trends, funnels, user paths, retention analysis, and segment user cohorts. Also, direct [SQL querying](/docs/product-analytics/sql) for power users.
+
+- 📺 **Session replays:** View exactly how users are using your site. Includes event timelines, console logs, network activity, and 90-day data retention.
+
+- 🧪 **A/B tests:** Experiment in your app with up to nine test variations and track impact on primary and secondary metrics. Auto-calculate test duration, sample size, and statistical significance.
+
+- 🚩 **Feature flags:** Rollout features safely with [local evaluation](/docs/feature-flags/local-evaluation) (for faster performance), JSON payloads, and instant rollbacks.
+
+### How does PostHog compare to Survicate?
+
+PostHog and Survicate have similar capabilities when it comes to surveys but differ on other features. PostHog includes entire features Survicate doesn't have, like product analytics, A/B testing, and session replays. 
+
+On the other hand, Survicate supports sending surveys on more platforms, such as mobile app, emails, and shareable-links. PostHog doesn't yet offer surveys on mobile – although all of its other features are available on apps.
+
+PostHog and Survicate both offer integrations so that you can send your survey data to your other tools. However, PostHog's integrations requires basic technical setup using [webhooks](/docs/webhooks), whereas Survicates integrations are all 1-click.
+
+<ComparisonTable column1="PostHog" column2="Survicate">
+  <ComparisonRow column1={true} column2={true} feature="Surveys" description="Collect and analyze feedback, run NPS and PMF"/>
+  <ComparisonRow column1={false} column2={true} feature="Multi-channel surveys" description="Send surveys in mobile apps, email, links" />
+  <ComparisonRow column1={true} column2={true} feature="Integrations" description="Send survey data to all your tools" />
+  <ComparisonRow column1={true} column2={false} feature="Session replay" description="Watch real users on your site; discover friction points" />
+  <ComparisonRow column1={true} column2={false} feature="Heatmaps" description="Visualize where users click and interact on your site" />
+  <ComparisonRow column1={true} column2={false} feature="Product analytics" description="Track trends, funnels, user paths, and retention" />
+  <ComparisonRow column1={true} column2={false} feature="A/B testing" description="Test changes and analyze their impact" />
+  <ComparisonRow column1={true} column2={false} feature="Feature flags" description="Deploy features safely with targeting and percentage rollouts" />
+  <ComparisonRow column1={true} column2={false} feature="Open source" description="Audit code, contribute to roadmap, and build integrations" />
+</ComparisonTable>
```

Comment:
I think the comparison table needs to go deeper on actual survey features given Survicate's focus on that. This feels a bit too self-serving to focus on the things we do, rather than honestly comparing our survey features.